### ISSUE TITLE
FrameDuration for each frame when using SpriteSheetAnimationFactory.Add()

### ISSUE THREAD

**Issue by Dante3085** [MonoGame-Extended/Monogame-Extended#694]
I followed the [MonoGame.Extended tutorial](http://docs.monogameextended.net/Features/Animations/)  for creating animations and couldn't find a way to set the frameDuration for an individual frame. It seems like there is just one frameDuration for the whole animation like in the following piece of code...

```
animFactory.Add("idle", new SpriteSheetAnimationData(new int[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 }, frameDuration: .1f, isLooping: true));
```

I created this [animation ](https://imgur.com/a/9MjhrcY) in Aseprite.
The first frame has a slow startup of 300ms and all other frames have 100ms delay. It seems like this is not possible with the animation tools MonoGame.Extended provides. I understand if it's too much of a change to the animation system, but it would improve the expressiveness of animations.

**Comment by lithiumtoast** [comment]
@Dante3085 Yes, I agree, I think it would be a good idea to have individual frame durations. I guess it wasn't an expected thing before. Do you have the animation file for Aseprite I could use for piloting this?

**Comment by Dante3085** [comment]
@lithiumtoast Thanks for taking the time! 
Yes, I uploaded the exported spritesheet and the original .aseprite file to wetransfer. You can download them over this [link](https://we.tl/t-zq1aFUb2T9).

**Comment by AristurtleDev** [comment]
Will be looking into implementing this as part of 4.1 when Aseprite integration is added

**Comment by AristurtleDev** [comment]
Closing this as individual frame timing was added during the sprite rework #897
